css/styles.css

```diff
@@ -0,0 +1,154 @@
+.navbar {
+  text-align: center;
+  background-image: url('https://images.unsplash.com/photo-1500754088824-ce0582cfe45f?w=500&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8NTJ8fDE5MjB4MTA4MCUyMHdhbGxwYXBlcnxlbnwwfHwwfHx8MA%3D%3D');
+  height: 400px;
+}
+
+h1 {
+  color: white;
+  padding-top: 150px;
+  font-size: 60px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  min-height: 300px;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+  text-align: center;
+}
+
+h2 {
+  font-size: 30px;
+  color: white;
+}
+
+.column-fifth {
+  flex-basis: calc(100% / 5);
+  margin: 30px;
+  text-align: center;
+}
+
+.search-character {
+  width: 200px;
+  height: 30px;
+  padding: 10px;
+  border-radius: 8px;
+  margin: 20px;
+}
+
+button {
+  margin: 20px;
+  height: 30px;
+  width: 80px;
+  border-radius: 8px;
+}
+
+.hidden {
+  display: none;
+}
```

Comment:
Should have `display: none !important` to make sure this always "wins" in the cascade